website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -49,6 +49,12 @@ millions of leases in an automated way, it is recommended to space out the creat
   concept of roles (such as `/auth/approle/`), this will make the quota restrict login
   requests to that mount that are made with the specified role. The request will fail if
   the auth mount does not have a concept of roles, or `path` is not an auth mount.
+- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
+all child namespaces of the specified path namespace will inherit this quota. The request
+will fail if `inheritable` is set to true, while the `path` has a value that is not a namespace mount,
+or if a value for `role` is specified. All global quotas are inheritable by default. All namespace quotas
```

Comment:
I think I'd suggest rewording this for clarity, especially as we made the decision that root quotas can't be made uninheritable:
"All global quotas are inheritable by default. All namespace quotas are not inheritable by default."
-> 
"Root namespace quotas are inheritable by default and this cannot be changed, thereby making them global in reach. All other namespace quotas are not inheritable by default."